components/scream/src/share/grid/point_grid.cpp

```diff
@@ -121,6 +123,36 @@ create_point_grid (const std::string& grid_name,
 
   grid->set_dofs(dofs_gids);
 
+  using device_type       = DefaultDevice;
```

Comment:
For consistence safety, it might be better to get these types (except the constants) from the grid type itself:

```
using geo_view_type = PointGrid::geo_view_type;
using KT                    = PointGrid::kokkos_types;
```
They are defined [here](https://github.com/E3SM-Project/scream/blob/e3a2be3e60c78610121cd91eaa4765dc79cdb500/components/scream/src/share/grid/abstract_grid.hpp#L40).

Also, KT and kokkos_types are redundant here.